corpus/config/migrations/0002_society_description_society_name.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 4.2.4 on 2023-11-13 17:31
```

Comment:
Are we pushing migrations also?

Reply:
Yes. It's good practice to do so. In case we deploy a version of the DB with a particular field and later decide to remove that field, migrations will ensure the removal is done in a clean manner.

[Django Docs on migrations](https://docs.djangoproject.com/en/4.2/topics/migrations/)